x-pack/plugins/apm/common/ml_job_constants.ts

```diff
@@ -13,10 +20,32 @@ export function getMlJobId(serviceName: string, transactionType?: string) {
   return `${getMlPrefix(serviceName, transactionType)}high_mean_response_time`;
 }
 
+export function getMlJobServiceName(jobId: string) {
+  const parts = jobId.split('-');
+  parts.splice(parts.length - 2, 2);
+  return parts.join('-');
+}
```

Comment:
I think this method should be co-located with the other ML job contants in `x-pack/plugins/apm/common/ml_job_constants.ts` 

btw. WDYT about this one-liner?
```suggestion
export function getMlJobServiceName(jobId: string) {
  return jobId.split('-').slice(0, -2).join('-');
}
```

Reply:
Updated in 3bbd7dd.